vaadin-spring/src/main/java/com/vaadin/flow/spring/VaadinApplicationConfiguration.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2000-2020 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.spring;
+
+import org.springframework.context.ApplicationContextAware;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import com.vaadin.flow.spring.SpringLookupInitializer.SpringApplicationContextInit;
+
+/**
+ * Vaadin Application Spring configuration.
+ * <p>
+ * Registers a default {@link ApplicationConfigurationFactory} for Vaadin web
```

Comment:
`ApplicationConfigurationFactory` does not exist in 12.4.

Reply:
Done